packages/portal-proto/src/features/biospecimen/Biospecimen.test.tsx

```diff
@@ -2,6 +2,10 @@ import { render } from "@testing-library/react";
 import { Biospecimen } from "./Biospecimen";
 import * as func from "@gff/core";
 
+jest.mock("next/router", () => ({
+  useRouter: jest.fn(),
+}));
```

Comment:
Did something change in the newer versions of nextjs that requires this? Or has this test always been broken?

Reply:
This test was not broken before. I noticed that after the upgrade this test failed because of [this](https://nextjs.org/docs/messages/next-router-not-mounted) reason. When looking at the file I noticed that `useRouter` had never been mocked before.